pkg/dbmate/migration.go

```diff
@@ -39,9 +40,9 @@ func (m *Migration) Parse() (*ParsedMigration, error) {
 
 // ParsedMigration contains the migration contents and options
 type ParsedMigration struct {
-	Up          string
+	Up          []string
 	UpOptions   ParsedMigrationOptions
```

Comment:
I think `UpOptions` and `DownOptions` need to be turned into arrays, as well, as each migration can have its own options.

Reply:
I went a bit other way and implemented file sections